os/net/netmgr/netdev_lwip.c

```diff
@@ -617,12 +617,14 @@ static err_t _netif_loopif_init(struct netif *netif)
 
 	netif->name[0] = 'l';
 	netif->name[1] = 'o';
+#if LWIP_HAVE_LOOPIF
```

Comment:
I think these common functionality is for loopback interface , why do we need to remove the condition?